playbook/app/pb_kits/playbook/pb_body/_body.scss

```diff
@@ -42,6 +42,10 @@
     @include body_truncate;
   }
 
+  &[class*=_text_hover]:hover {
+    color: $white !important;
+  }
```

Comment:
I'm not sure I understand this addition, and the need for the important tag. Is this needed?

Reply:
Talked to Jasper and we decided to move this to the `hover` global prop.